### PR TITLE
fix get forum channel id from .env

### PR DESCRIPTION
**Pull request by berkingurcan**
- need to add POST_CHANNEL_ID to .env.local
- fixed debug logs from info

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -22,5 +22,5 @@ export const discordConfig = {
 
 export const version = package_json.version;
 
-export const POST_CHANNEL_ID = "1245319554201157724"
+export const POST_CHANNEL_ID = process.env.POST_CHANNEL_ID.toString() || ""
```
Comment (johnmarcou) [inline src/config.js:25]: **question:** Can the application run without channel ID? Should that be optional?

File: src/summarizer.js [hunk 2]
```diff
@@ -24,13 +24,13 @@ const main = async () => {
   await redisClient.connect();
 
   while (true) {
-    log.info("checking surveys for updates...");
+    log.debug("checking surveys for updates...");
```
Comment (johnmarcou) [inline src/summarizer.js:27]: **praise:** That's great! The logs will look cleaner 👍

File: src/config.js (hunk 1 above)
Comment (berkingurcan) [reply src/config.js:25]: Yes, can be optional. Discord bots continues to run after throwing API req errors.

**Review by johnmarcou — APPROVED** [review]
LGTM!

**Pull request merged**